launch_yaml/launch_yaml/parser.py

```diff
@@ -51,3 +53,8 @@ def load(
         finally:
             if didopen:
                 fileobj.close()
+
+    @classmethod
+    def get_file_extensions(cls) -> Set[Text]:
```

Comment:
@christophebedard we should use these extensions on `Parser.load()` as well.

Reply:
done as part of e526f9cbdf04a7ce20dea2e37e4234d68b5e0253